doc/source/indexing.rst

```diff
@@ -249,6 +249,14 @@ new column.
 If you are using the IPython environment, you may also use tab-completion to
 see these accessible attributes.
 
+You can also assign a ``dict`` to a row of a ``DataFrame``:
```

Comment:
Can you maybe specify here that the dict keys are matched with the column names?